fbinternal/teravm/fabfile_teravm_conf.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
As a config example I would put all variable that have to be input inside "<QUICK_DESCRIPTION>"